templates/featured.php

```diff
@@ -4,12 +4,12 @@
       <div class="govuk-grid-column-full">
         <article <?php post_class('featured') ?>>
           <div class="featured-wrapper">
-            <h2 class="entry-title visible-tablet"><a href="<?php the_permalink() ?>"><?php the_title() ?></a></h2>
+            <h2 class="govuk-heading-m entry-title visible-tablet"><a href="<?php the_permalink() ?>"><?php the_title() ?></a></h2>
```

Comment:
There seems to be a minor bug with this. The `visible-tablet` class is meant to hide this heading completely on desktop, but the spacing is still being rendered, creating a gap at the top of the `<div class="featured-wrapper">`.

Correct rendering (on `master`):

<img width="990" alt="Screenshot 2020-04-23 at 10 56 28" src="https://user-images.githubusercontent.com/370665/80086075-18044680-8551-11ea-994c-8c31bdb26ae0.png">

Rendering on this branch:

<img width="974" alt="Screenshot 2020-04-23 at 10 57 55" src="https://user-images.githubusercontent.com/370665/80086213-4c780280-8551-11ea-8eb4-98acb6ac2962.png">

You can see this on the homepage of a blog, if you tick the "Featured post" box when editing a post to make it show up at the top of the homepage.

Reply:
@isratc I think the easiest way to fix this might be to put an `!important` on the `display:none` here: https://github.com/dxw/govuk-blogs/blob/f5bffb063f9da516c2c829c62594ef91732fcc0f/assets/css/common/common.scss#L38